README.md

```diff
@@ -444,6 +501,8 @@ select * from apilinks;
 
 ![image](https://user-images.githubusercontent.com/86580534/135923860-d10a2323-100c-446e-bec3-6010cca2ba8b.png)
 
+<br>
+
 ## Use a database server running as a container
 
```

Comment:
Comment for the renamed file below - good to see renaming the catch4 -> catchment_polygon. In the future you may want to also consider making them geojson files instead of shapefiles. This is a more modern file format.

Reply:
This isn't needed in my view for this PR but something I think it would be worth doing for any future shapefiles you create or next time you interact with these files. You can use geopandas to easily convert to geojson files.